smoketests/index.js

```diff
@@ -11,6 +11,8 @@ const tests = [
 
 (async () => {
   let isAllPassed = true;
+  const passResults = [];
+  const failResults = [];
 
   for await (const test of tests) {
     console.log(`\nRUN  ${test.name}`);
```

Comment:
Can we make this bold and colorful using chalk, to make it more prominent?

Reply:
Jest output in CI is uncolored but is readable as it gives summary in the end (see screenshot below). So, I think no need.

![image](https://user-images.githubusercontent.com/15812317/172995753-1ec09a2e-fcc3-44e5-a4b8-550d2a8b890d.png)

Reply:
Actually, so many output logs confused me at first glance

<img width="958" alt="Screenshot 2022-06-10 at 11 07 51 AM" src="https://user-images.githubusercontent.com/46647141/172997791-53dbb642-02b8-48df-9537-3344c0ee2bb2.png">

Reply:
> I think we can update this line:

No we should not. We disable colors by default because it breaks snapshots because of uneven color unicode that color text in different OS. This would potentially break many tests.

Reply:
> Actually, so many output logs confused me at first glance

That is why I am adding summary now 😄

Reply:
Oh! got it. I was looking at the wrong logs.

>This would potentially break many tests.

I meant update something like this only for smoketest - `if (!expect.getState().testPath.includes("colors.test.js") || !expect.getState().testPath.includes("smoketest")) { `


But yes, not needed. I was looking at the wrong logs.